Libplanet.Tests/Action/ActionEvaluatorTest.cs

```diff
@@ -708,7 +710,7 @@ public void EvaluateTxResultThrowingException()
             var actionEvaluator = new ActionEvaluator<ThrowException>(
                 policyBlockAction: null,
                 blockChainStates: NullChainStates<ThrowException>.Instance,
-                trieGetter: null);
+                trieGetter: null, genesisHash: null);
```

Comment:
For consistency, please give a line to each argument.

```suggestion
                trieGetter: null,
                genesisHash: null);
```